source/internal/_reduce.js

```diff
@@ -37,20 +37,25 @@ function _methodReduce(xf, acc, obj, methodName) {
 var symIterator = (typeof Symbol !== 'undefined') ? Symbol.iterator : '@@iterator';
 
 export default function _reduce(fn, acc, list) {
+  var reduced = true;
```

Comment:
We should change the name of this variable. It mirrors a public function, there's no indication that it's a predicate, and it doesn't actually say whether `reduced` has been called (I think). Maybe something like `shouldUseReduced`?

Reply:
My vote is on `useReduced`.